<launch>
  <arg name="robot_name"         	 default="dasom"/>
  <arg name="device_name"                default="/dev/ttyUSB0"/>
  <arg name="baud_rate"                  default="1000000"/>

  <arg name="scan_range"                 default="10"/>

  <arg name="profile_velocity"           default="200"/>
  <arg name="profile_acceleration"       default="50"/>

  <arg name="position_p_gain"		 default="0"/>
  <arg name="position_i_gain"		 default="0"/>
  <arg name="position_d_gain"		 default="0"/>
  <arg name="feedforward_1st_gain"	 default="0"/>
  <arg name="feedforward_2nd_gain"	 default="0"/>

  <param name="robot_name"               value="$(arg robot_name)"/>
  <param name="device_name"              value="$(arg device_name)"/>
  <param name="baud_rate"                value="$(arg baud_rate)"/>

  <param name="scan_range"               value="$(arg scan_range)"/>

  <param name="profile_velocity"         value="$(arg profile_velocity)"/>
  <param name="profile_acceleration"     value="$(arg profile_acceleration)"/>

  <param name="position_p_gain"		 value="$(arg position_p_gain)"/>
  <param name="position_i_gain"		 value="$(arg position_i_gain)"/>
  <param name="position_d_gain"		 value="$(arg position_d_gain)"/>
  <param name="feedforward_1st_gain"	 value="$(arg feedforward_1st_gain)"/>
  <param name="feedforward_2nd_gain"	 value="$(arg feedforward_2nd_gain)"/>

  <node name="position_control" pkg="dynamixel_workbench_controllers" type="position_control" required="true" output="screen"/>
</launch>
